src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -12,25 +12,38 @@
     :eventValue {:action action}}})
 
 (def ^:const app-started-event "app-started")
-(def ^:const navigate-to-create-profile-event "navigate-to-create-profile")
-(def ^:const communities-tab-clicked "communities-tab-clicked")
-(def ^:const wallet-tab-clicked "wallet-tab-clicked")
-(def ^:const chats-tab-clicked "chats-tab-clicked")
+
+(def ^:const view-ids-to-track
+  [;; Tabs
+   :communities-stack
+   :chats-stack
+   :wallet-stack
+
+   ;; Onboarding
+   :screen/onboarding.intro
+   :screen/onboarding.new-to-status
+   :screen/onboarding.sync-or-recover-profile
+   :screen/onboarding.enter-seed-phrase
+   :screen/onboarding.create-profile
+   :screen/onboarding.create-profile-password
+   :screen/onboarding.enable-biometrics
+   :screen/onboarding.generating-keys
+   :screen/onboarding.enable-notifications
+   :screen/onboarding.sign-in-intro
+   :screen/onboarding.sign-in
+   :screen/onboarding.syncing-progress
+   :screen/onboarding.syncing-progress-intro
+   :screen/onboarding.syncing-results
+   :screen/onboarding.welcome])
 
 (defn track-view-id-event
   [view-id]
-  (case view-id
-    :communities-stack (user-journey-event communities-tab-clicked)
-    :chats-stack       (user-journey-event chats-tab-clicked)
-    :wallet-stack      (user-journey-event wallet-tab-clicked)
-    nil))
+  (when (some #{view-id} view-ids-to-track)
```

Comment:
Probably not a problem @shivekkhurana, but `case` does a constant time dispatch, so it's much faster than `some` used with sets. Ideally in the future we could have a macro to make this more convenient but have the least amount of overhead possible.

But totally looks good for now!

Reply:
My idea was to have a list of screen ids that we can track.

But I understand that `some` is not the best. So I have used `contains?` with a Set which will have an O(1) complexity in the average case.